### PR TITLE
UX: pretty up staff color block margins

### PR DESCRIPTION
**Pull request by MarcRez33**

### CODE REVIEW

**Comment by tgxworld** [comment]
@MarcRez33 Thank you for the contribution. To assist us with the review, can you provide a before/after screenshot related to your change in this PR?

**Comment by tgxworld** [comment]
ping @discourse/designers

**Comment by MarcRez33** [comment]
They are on meta: https://meta.discourse.org/t/staff-color-post-notice-css-improvements/232093?u=marcp

Just ignore the rounded borders, that is custom css.

**Review by Chapoi — DISMISSED** [review]

File: app/assets/stylesheets/mobile/topic-post.scss [hunk 1]
```diff
@@ -430,3 +430,9 @@ span.highlighted {
   opacity: 100%;
   margin-bottom: 1rem;
 }
+
+// fix staff color margins
+.moderator .regular > .cooked {
+  padding-top: 1em;
+  padding-left: 1em;
+}
```
Comment (Chapoi) [inline app/assets/stylesheets/mobile/topic-post.scss:438]:
```suggestion
}

```

**Review by awwsssssssss — COMMENTED** [review]
Aprove

**Comment by pmusaraj** [comment]
@chapoi @awesomerobot can you decide one way or another with this PR, please?

**Pull request closed without merging**